mlir/lib/Dialect/Rock/IR/RockGemmWrapperInterface.cpp

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "mlir/Dialect/Rock/IR/RockGemmWrapperInterface.h"
+#include "mlir/Dialect/Rock/IR/Rock.h"
```

Comment:
Why'd this need to change?

Reply:
Now we use GeneralGemmParamsAttr defined in RockAttr.td, only including RockGemWrapperInterface.h can't find it.
Rock.h includes header files in correct order.

Reply:
Fair enough, makes sense